src/helpers/MiscFunctions.cpp

```diff
@@ -722,6 +722,14 @@ int64_t configStringToInt(const std::string& VALUE) {
     return std::stoll(VALUE);
 }
 
+Vector2D configStringToVector2D(const std::string& VALUE) {
+    const auto SIZESTR  = VALUE.substr(VALUE.find(' ') + 1);
```

Comment:
brother. With all due respect, are you ok??

This function still takes only `x y`, you moved the logic OUTSIDE, not IN THIS FUNCTION.

Furthermore, why are you even passing a bullshit argument here as the first arg??

Ugh, I'll just fix this bs myself.

Reply:
>you moved the logic OUTSIDE, not IN THIS FUNCTION

It has to be outside, because how else would the window rule handle the extra arg?

>This function still takes only x y

It doesn't? Technically it could even take any additional character that isn't a comma or number right after it, like `500. 500` or even `500dsflskdfmlk 500sdnfkljfnslk`

Your patch didn't even work, because it interprets the `y` arg as the window arg itself:

`[LOG] Window rule minsize 1000 ->  500, class:foot matched`

when it should be:

`[LOG] Window rule minsize 1000 500 ->  class:foot matched`

Reply:
you're piping `500hwufbgufrhbgeru` into `std::stoll`, that doesn't throw?

> Your patch didn't even work, because it interprets the y arg as the window arg itself:

simple fix

Reply:
>you're piping 500hwufbgufrhbgeru into std::stoll, that doesn't throw?

no, only if there are non-numbers *before* the number like `alfdkjgfdlk500`

Reply:
that's... cringe.

Still:
 - why does it take a useless word at the beginning? (the func wont parse `500 500` it will take `garbage 500 500` and extract the `500 500`)
 - why the try catch block calling it twice?

Reply:
true, changed